jbpm-db-scripts/src/main/resources/db/ddl-scripts/mysqlinnodb/mysql5-cluster-jbpm-schema.sql

```diff
@@ -0,0 +1,854 @@
+    create table Attachment (
+        id bigint not null auto_increment,
+        accessType integer,
+        attachedAt datetime(6),
+        attachmentContentId bigint not null,
+        contentType varchar(255),
+        name varchar(255),
+        attachment_size integer,
+        attachedBy_id varchar(255),
+        TaskData_Attachments_Id bigint,
+        primary key (id)
+    );
+
+    create table AuditTaskImpl (
+        id bigint not null auto_increment,
+        activationTime datetime(6),
+        actualOwner varchar(255),
+        createdBy varchar(255),
+        createdOn datetime(6),
+        deploymentId varchar(255),
+        description varchar(255),
+        dueDate datetime(6),
+        name varchar(255),
+        parentId bigint not null,
+        priority integer not null,
+        processId varchar(255),
+        processInstanceId bigint not null,
+        processSessionId bigint not null,
+        status varchar(255),
+        taskId bigint,
+        workItemId bigint,
+        lastModificationDate datetime(6),
+        observation varchar(255),
```

Comment:
Please remove this field, there is no such field in the rest of the scripts for this particular table

Reply:
removed